playbooks/spark-integration-test-minikube-k8s/run.yaml

```diff
@@ -0,0 +1,29 @@
+- hosts: all
+  become: yes
+  roles:
+    - role: install-docker-ce
+      docker_ce_version: 18.06
+      docker_vcgroupdriver: "cgroupfs"
+    - create-single-k8s-cluster-with-minikube
+  tasks:
+    - name: Run integration tests of Spark deployed on k8s cluster
```

Comment:
I think `Run integration tests of Spark with k8s cluster manager` is more exact describe.